rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -73,6 +74,28 @@ def __call__(
         ...
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=True)
+class DecodingRules():
+    address_mappings: dict[ChecksumEvmAddress, tuple[Any, ...]]
+    event_rules: list[EventDecoderFunction]
+    token_enricher_rules: list[Callable]
+    post_decoding_rules: list[tuple[Callable, int]]
```

Comment:
nitpick: From a logical perspective I would expect priority to come first and the callable second, in a tuple